src/biokbase/narrative/jobs/job.py

```diff
@@ -67,30 +67,28 @@ def _attr_to_ee2(attr):
         return attr
 
 
+def get_dne_job_state(job_id, output_state=True):
+    state = {"job_id": job_id, "status": "does_not_exist"}
+    if output_state:
+        state = {"state": state}
+    return state
+
+
 class Job(object):
     _job_logs = list()
-    _const_state = None  # for original/constant attributes
-    _last_state = None  # for changing attributes
+    _acc_state = None  # accumulates state
```

Comment:
Replace my previous usage of a `_const_state` and `_last_state` with `_acc_state` which accumulates the changes